tests/test_frame_selection.py

```diff
@@ -101,21 +124,50 @@ def test_remove_all():
     tt_rms = 0
     for frame in default_dataset:
         frame.ext_hdr['RESZ2RMS'] = tt_rms
+        frame.ext_hdr['RESZ3RMS'] = tt_rms
         tt_rms += 1
     # add overexp
     default_dataset[1].ext_hdr['OVEREXP'] = True
 
     # keep only 1 frame
     # bpix_frac removes index 0
     # overexp removes index 1
-    # tt_thres removes indicies 3,4
-    pruned_dataset = frame_select(default_dataset, bpix_frac=0.1, overexp=True, tt_thres=2.5)
+    # tt_rms_thres removes indicies 3,4
+    pruned_dataset = frame_select(default_dataset, bpix_frac=0.1, overexp=True, tt_rms_thres=2.5)
     assert len(pruned_dataset) == 1
 
     # removes all frames
     with pytest.raises(ValueError):
-        pruned_dataset = frame_select(default_dataset, bpix_frac=0.1, overexp=True, tt_thres=1.5)
+        pruned_dataset = frame_select(default_dataset, bpix_frac=0.1, overexp=True, tt_rms_thres=1.5)
 
+def test_marking():
```

Comment:
doesn't this have to be added to the bottom of the file in `if __name__ == "__main__"`?